pkgs/tools/misc/ttyplot/default.nix

```diff
@@ -1,21 +1,20 @@
-{ lib, stdenv, fetchFromGitHub, ncurses }:
+{ lib, stdenv, fetchFromGitHub, ncurses, pkg-config }:
 
 stdenv.mkDerivation rec {
   pname = "ttyplot";
-  version = "1.5.2";
+  version = "1.6.1";
 
   src = fetchFromGitHub {
     owner = "tenox7";
     repo = "ttyplot";
     rev = version;
-    sha256 = "sha256-BYMdGNDl8HUin1Hu4Fqgx305a/tTt1fztqlT2vDeTh8=";
+    hash = "sha256-SQ5keCcwzQsSxfSevQwRa1eNf+8JXsrh1vljehI4tPc=";
   };
 
-  buildInputs = [ ncurses ];
-
-  buildPhase = ''
-   ${stdenv.cc}/bin/cc ./ttyplot.c -lncurses -o ttyplot
-  '';
+  buildInputs = [
+    pkg-config
+    ncurses
+  ];
```

Comment:
@Lassulus now that the build phases uses `make` — great! —, the install phase probably should do too, in particular to also install the man page (which has a ton of non-trivial examples). There three relevant lines from the `Makefile` are:

```
MANPREFIX ?= $(PREFIX)/man
```
…and…
```
        install -d $(DESTDIR)$(MANPREFIX)/man1
        install -m644 ttyplot.1 $(DESTDIR)$(MANPREFIX)/man1
```

Reply:
@Lassulus PS: `licenses.unlicense;` does not look like Apache 2. Is that intentional?

Reply:
ah, I guess back in the day there was no license yet when I packaged it